### ISSUE TITLE
Content to merge into this page or wiki

### ISSUE THREAD

**Issue by rweigel** [hapi-server/uri-templates#2]
The spec https://github.com/hapi-server/uri-templates/wiki/Specification has three broken links associated

The first two are

# http://tsds.org/uri_template.jjt
# http://tsds.org/Recommended_file_and_data_collection_naming_practices

The content of the above two pages should be moved here.

There is also a reference to a file that Jon may have tried to upload and I can't find anywhere
# uri-templates-2015-10-09.tgz

**Comment by rweigel** [comment]
What resource specifically?

**Comment by jbfaden** [comment]
The uri_template.jjt file, which I imagine was the syntax for valid URI templates.  I'd be interested in looking at that.  (I didn't appreciate syntaxes like that back then.  I've been playing with them a lot more over the past couple of years and see better why this was useful.

**Comment by rweigel** [comment]
This was in a tar file Jon posted. Neither he or I could find it.

**Comment by rweigel** [comment]
Yes, and I would modify http://autoplot.org/uri_templates to point to github.

**Comment by jbfaden** [comment]
Also this has a dead link to https://github.com/hapi-server/uri-templates/wiki/Specification#full-list-of-field-codes .
